cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class PaymentStepPage {
+    private payButton: string;
+    private confirmOrder: string;
```

Comment:
Try changing to something claryfing that this refers to the text, like:
```suggestion
    private confirmOrderMessage: string;
```

Reply:
done @danielgalvis98